### PR TITLE
OCPBUGS-15730: fixes typo in kind parameter of  yaml file

### PR DESCRIPTION
**Pull request by JoeAldinger**
OCPBUGS#[OCPBUGS-15730]: fixes typo in kind parameter of yaml file

Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OCPBUGS-15730

Link to docs preview:
https://62346--docspreview.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/configuring-egress-firewall-ovn.html#configuringNodeSelector-example_configuring-egress-firewall-ovn

QE review:
- [x] QE has approved this change.

Additional information:
@huiran0826 PTAL for QE feedback.

### CODE REVIEW

**Review by skopacz1 — APPROVED** [review]
LGTM! However I'm new to reviewing, so please wait until @adellape approves my review before proceeding. Thanks!

**Review by adellape — COMMENTED** [review]
@skopacz1 Peer review LGTM 👍

**Pull request merged**
